packages/core/src/renderer/components/workloads-pods/columns/pods-cpu-usage-column.injectable.tsx

```diff
@@ -0,0 +1,38 @@
+/**
+ * Copyright (c) Freelens Authors. All rights reserved.
+ * Copyright (c) OpenLens Authors. All rights reserved.
+ * Licensed under MIT License. See LICENSE in root directory for more information.
+ */
+
+import { podListLayoutColumnInjectionToken } from "@freelensapp/list-layout";
+import { getInjectable } from "@ogre-tools/injectable";
+import React from "react";
+import podStoreInjectable from "../../workloads-pods/store.injectable";
+import { COLUMN_PRIORITY } from "./column-priority";
+
+const columnId = "cpuUsage";
+
+export const podsUsedCpuColumnInjectable = getInjectable({
+  id: "pods-cpu-usage-column",
+  instantiate: (di) => ({
+    id: columnId,
+    kind: "Pod",
+    apiVersion: "v1",
+    priority: COLUMN_PRIORITY.CPU_USAGE,
+    content: (pod) => {
+      const podStore = di.inject(podStoreInjectable);
+      const metrics = podStore.getPodKubeMetrics(pod);
+      const cpuUsage = metrics.cpu
+      
+      return <span>{cpuUsage === 0 ? "N/A" : cpuUsage.toFixed(3)}</span>;
```

Comment:
I'm wondering whether it's appropriate to show N/A when the CPU value is 0, given that 0 is the default value.
Wouldn't it make more sense to show N/A only when the metric server is unavailable or the data can't be retrieved?

For reference, memory currently displays N/A when the value is 0, so this change was made for consistency.

Reply:
Hi @HanCheo, I'd expect `N/A` to be shown only when the metrics server is truly unavailable. Otherwise, we should print out the value returned by it, whatever it may be

Reply:
Yep I'm really agree. 
but... currently, getPodKubeMetrics returns { cpu: 0, memory: 0 } when no metrics are found.
However, returning 0 can be misleading, as it implies that the metrics exist and the usage is simply zero.
It would be more appropriate to return something like "N/A" to explicitly indicate that no data is available.
This helps distinguish between "no usage" and "no metrics."

And dosen't need compare cpuUsage === 0 

```ts
  getPodKubeMetrics(pod: Pod) {
    const containers = pod.getContainers();
    const empty = { cpu: 0, memory: 0 };  // <<-- here
    const metrics = this.kubeMetrics.find((metric) => {
      return [metric.getName() === pod.getName(), metric.getNs() === pod.getNs()].every((v) => v);
    });

    if (!metrics) return empty;
```

Reply:
And Memory ByteToUnits code. return "N/A" when <= 0.

```
export function bytesToUnits(bytes: number, { precision = 1 }: BytesToUnitesOptions = {}): string {
  if (bytes <= 0 || isNaN(bytes) || !isFinite(bytes)) {
    return "N/A";
  }
```

Reply:
I change it!